packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.2.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/V3.1.1...flutter-3.2.0) (2022-06-22)
+
+### Bug Fixes
+* Old version of connectivity_plus package ([#717](https://github.com/parse-community/Parse-SDK-Flutter/issues/717))
```

Comment:
Could you rewrite the changelog to:

```
### Bug Fixes

* old version of `connectivity_plus package` ([#717](https://github.com/parse-community/Parse-SDK-Flutter/issues/717))
* dependency `package_info_plus` does not work in web ([#714](https://github.com/parse-community/Parse-SDK-Flutter/issues/714))
* missing plugin exception, no implementation found for method `getAll` ([#712](https://github.com/parse-community/Parse-SDK-Flutter/issues/712))
* query fails for subclassed `ParseObject` in `ParseRelation` ([#697](https://github.com/parse-community/Parse-SDK-Flutter/issues/697))
* unhandled exception in `ParseRelation`, type `ParseObject` is not a subtype of type ([#696](https://github.com/parse-community/Parse-SDK-Flutter/issues/696))
* error in progress callback ([#679](https://github.com/parse-community/Parse-SDK-Flutter/issues/679))
* incorrect return type when calling `first()` ([#661](https://github.com/parse-community/Parse-SDK-Flutter/issues/661))
* error in `ParseLiveListWidget` when enabling `lazyloading` ([#653](https://github.com/parse-community/Parse-SDK-Flutter/issues/653))
```

These entries are not part of the changelog:
```
* General improvements
* Updated dependencies
```

And can you please split up the changes per package:
- a change that only happened in the dart package must only be in the dart changelog
- a change that only happened in the flutter package must only be in the flutter changelog
- a change that happened in both packages must be in the dart and flutter changelogs

Reply:
Information about updating dependencies is required.
I agree that within the Bug fix section it is not correct.
But the information must be indicated in the changelog

In a next version, I can do the split changelog per package.
Release this version and the next ones we will do as requested.

Reply:
>  agree that within the Bug fix section it is not correct.

OK, let's add this at the top before `Bug Fixes`:

```
### Features

- upgrade various dependencies
```

There are several issues here, not only are the changelog links pointing to issues instead of PRs, there is also no issue that the entry `upgrade various dependencies` can point to. This PR contains multiple fixes, which we won't do in the future anymore to avoid this situation.

> In a next version, I can do the split changelog per package.

We are already doing this since the last 3 releases, this is already a hard requirement for PRs.

Reply:
Could you please remove the "features" entry from the changelog and add the "breaking changes" entry?

Reply:
It was made in the Flutter package. In the Dart package it is not necessary, as there is no package with MAJOR update.